<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateData</class>
 <widget class="QWidget" name="UpdateData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Informations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,5,1">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QLineEdit" name="websiteNameLineEdit">
         <property name="placeholderText">
          <string>Website</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="emailLineEdit">
         <property name="placeholderText">
          <string>E-mail</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="websiteLinkLineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Link</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lastNameLineEdit">
         <property name="placeholderText">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="mainDeviceLineEdit">
         <property name="placeholderText">
          <string>Main Device</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="securityQuestionLineEdit">
         <property name="placeholderText">
          <string>Security Question 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="answerLineEdit">
         <property name="placeholderText">
          <string>Answer 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="purposeOfUseLineEdit">
         <property name="placeholderText">
          <string>Purpose of Use</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="firstNameLineEdit">
         <property name="placeholderText">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="linkedWebsiteLineEdit">
         <property name="placeholderText">
          <string>Linked Website</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="phoneNumerLineEdit">
         <property name="placeholderText">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="usernameLineEdit">
         <property name="placeholderText">
          <string>UserName</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="secondEmailLineEdit">
         <property name="placeholderText">
          <string>Second E-mail</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDateEdit" name="birthDateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="notePlainTextEdit">
       <property name="placeholderText">
        <string>Notes</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="specialDigitsCheckBox">
         <property name="text">
          <string>Special Digitis</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="numbersCheckBox">
         <property name="text">
          <string>Numbers</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upperCaseCheckBox">
         <property name="text">
          <string>Upper Case</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lowerCaseCheckBox">
         <property name="text">
          <string>Lower Case</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="lengthPasswordSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generatePasswordPushButton">
         <property name="text">
          <string>Generate Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updatePushButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
